services/provider/server/server.go

```diff
@@ -566,17 +567,20 @@ func decodeAndValidateTicket(ticket string, pubKey *rsa.PublicKey) (*services.On
 // FulfillStorageClaim RPC call to create the StorageClaim CR on
 // provider cluster.
 func (s *OCSProviderServer) FulfillStorageClaim(ctx context.Context, req *pb.FulfillStorageClaimRequest) (*pb.FulfillStorageClaimResponse, error) {
+	klog.Infof("FullfillStorageClaim to create Storageclaim %v", req.StorageClaimName)
```

Comment:
Since the function returns with error that code is not implemented, does this info make sense?